.gitignore

```diff
@@ -24,3 +24,5 @@ ios/Converse.xcworkspace/xcshareddata/swiftpm/Package.resolved
 
 # Reassure output directory
 .reassure
+ios/preview-GoogleService-Info.plist
```

Comment:
firebase engineers have said that it is okay to have this in source control as none of these keys are sensitive

Reply:
on second thought it's okay to have this public but I'm going to remove it from source control